src/main/java/org/jabref/gui/mergeentries/newmergedialog/ThreeWayMergeView.java

```diff
@@ -47,7 +47,7 @@ public class ThreeWayMergeView extends VBox {
     public ThreeWayMergeView(BibEntry leftEntry, BibEntry rightEntry, String leftHeader, String rightHeader, GuiPreferences preferences) {
         this.preferences = preferences;
 
-        getStylesheets().add(ThreeWayMergeView.class.getResource("ThreeWayMergeView.css").toExternalForm());
+        getStylesheets().add(ThreeWayMergeView.class.getResource("/org/jabref/gui/Base.css").toExternalForm());
```

Comment:
Remove this line.

(And check if JabRef still looks good)

Reply:
@HoussemNasri can you take a look?